июн. 04, 2024 9:58:53 AM debug.logging.MainLogger logMessage
INFO: App initialized
июн. 04, 2024 9:58:55 AM debug.logging.MainLogger logSevere
SEVERE: Exception occurred!

org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 2.1
-> Applied to database : -612684510
-> Resolved locally    : -1769165319
Either revert the changes to the migration, or run repair to update the schema history.
Migration checksum mismatch for migration version 2.2
-> Applied to database : 577597601
-> Resolved locally    : -1007615497
Either revert the changes to the migration, or run repair to update the schema history.
Migration checksum mismatch for migration version 3
-> Applied to database : -92387719
-> Resolved locally    : 380343871
Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://rd.gt/3AbJUZE
	at org.flywaydb.core.Flyway.lambda$migrate$0(Flyway.java:169)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:210)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:156)
	at model.states.main.MainFSM.initialize(MainFSM.java:55)
	at Main.init(Main.java:24)
	at Main.main(Main.java:11)

июн. 04, 2024 10:00:12 AM debug.logging.MainLogger logMessage
INFO: App initialized
июн. 04, 2024 10:03:07 AM debug.logging.MainLogger logMessage
INFO: App initialized
